core/src/main/java/hudson/slaves/DumbSlave.java

```diff
@@ -66,7 +66,7 @@ public DumbSlave(@Nonnull String name, String remoteFS, ComputerLauncher launche
     }
 
     @Extension @Symbol({"dumb",
-            "slave"/*because this is in effect the canonical slave type*/})
+            "slave", "agent" /*because this is in effect the canonical slave type*/})
```

Comment:
not sure the comment still makes sense

Reply:
This doesn't create a conflict in the Declarative Pipeline with its use of agent or with other locations where "agent" is already used?